apollo-portal/src/main/resources/static/views/component/namespace-panel-master-tab.html

```diff
@@ -316,7 +316,7 @@
                                     <span ng-bind="config.item.comment | limitTo: 250"></span>
                                     <span ng-bind="config.item.comment.length > 250 ?'...' : ''"></span>
                                 </td>
-                                <td width="10%" ng-bind="config.item.dataChangeLastModifiedBy">
+                                <td width="10%" ng-bind="config.item.dataChangeLastModifiedByPreferredUsername + '(' + config.item.dataChangeLastModifiedBy + ')'">
```

Comment:
I think we also need to update the namespace-panel-branch-tab.html

Reply:
Besides, shall we also update the default user manage page to add the preferred user name?

![image](https://user-images.githubusercontent.com/837658/114297531-b7647580-9ae3-11eb-84a0-f7eec9ce4923.png)